static/app/views/performance/transactionSummary/transactionOverview/durationChart/content.tsx

```diff
@@ -77,32 +119,17 @@ function Content({
         }
       : undefined,
     yAxis: {
+      minInterval: durationUnit,
```

Comment:
This property does a lot here. With this change, because we have a fixed unit used in the yAxis, this property ensures that there is at least that duration between each tick. For example if the yAxis uses minutes, there will be at least one minute between each tick preventing weird rounding and duplicated yAxis ticks.

Reply:
Hmm, thinking about this approach, getting around this with a fixed interval might not necessarily be what we want, eg. if you have values in the range 1000-1200ms, the `minInterval` would therefore be a second? Would it just show ticks for 1 second and 2 seconds? It's useful to have the decimals if the range is small enough since you care about a 100ms increase if all your durations are in the 1000ms range.

I'm in and out in the next bit, can you confirm this will change the ticks to make them have less resolution when it's around the threshold of our duration units? (Confirm with a screenshot or something so if I'm out others can chime in?).

Reply:
I will look into it and think about it more then circle back! 

But My initial thoughts: That's a great point and I've thought of a similar case as well. We can still have a fixed unit but with decimal points.

Reply:
Here's an example between 1000-1200
BEFORE
<img width="926" alt="image" src="https://user-images.githubusercontent.com/44422760/179271769-a0cc0493-7fd6-47f4-8662-18005e55848b.png">

AFTER
<img width="924" alt="image" src="https://user-images.githubusercontent.com/44422760/179271710-fc63f75c-05f1-4020-8193-de173c1f060c.png">

Personally, I think what this PR brings is better then what we have currently but not ideal and we can just be "smarter" with our choice of units. For example use ms in this case
<img width="898" alt="image" src="https://user-images.githubusercontent.com/44422760/179272156-218fbcea-76be-4348-ac7e-e620754c6289.png">

Reply:
Implemented as discussed via slack